src/commands/server/start.ts

```diff
@@ -142,6 +142,13 @@ export default class Start extends Command {
       this.exit()
     }
 
+    if (flags.platform === 'minikube' && flags.installer && flags.installer.includes('minishift')) {
+      platformCheckTasks.add({
```

Comment:
If you want to add a task to the minikube `startTasks` you should do that in `minikube.ts` rather then here.

Reply:
I didn't want to add it in minikube `startTasks`

AFAIK Minikube startTasks are only for minikube. Here it's more to add a check independent of the runner. Only start() command has the global view of platform and installer

I think there is like a design failure BTW. We should register installer per platform instead of allowing installer globally

I could move it to minikube.ts but IMHO I kept the logic of the start task making the redirect to different implementation based on the flags

Reply:
I see and it makes sense. Then you can just fail without the need of adding a `platformCheckTask`.

Reply:
ok